go/vt/tableacl/tableacl.go

```diff
@@ -319,23 +316,3 @@ func GetCurrentACLFactory() (acl.Factory, error) {
 	}
 	return nil, fmt.Errorf("aclFactory for given default: %s is not found", defaultACL)
 }
-
-func checkHealth(acl *tableACL) error {
```

Comment:
was this unused functionality?

Reply:
ACL itself isn't used by most people. Also, I deleted this code because I found it untasteful, because it's not publishing healthcheck at all. It's just publishing what the current rules are. If someone wants it back, we can implement it the right way.